src/bun.js/api/server/ServerConfig.zig

```diff
@@ -34,6 +34,7 @@ sni: ?bun.BabyList(SSLConfig) = null,
 max_request_body_size: usize = 1024 * 1024 * 128,
 development: DevelopmentOption = .development,
 broadcast_console_log_from_browser_to_server_for_bake: bool = false,
+enable_uncaught_exception_reporting_from_browser_to_terminal_for_bake: bool = true,
```

Comment:
i wonder if it could be an enum instead as console.log inherently also includes uncaught errors